### ISSUE TITLE
Loading jQuery from index.html (SystemJS)

### ISSUE THREAD

**Issue by bitf09a013** [evansmwendwa/ng2-daterangepicker#4]
Firstly thank you for the great work! It all works great if I install the modules using NPM. But for some reason, I am working in an application which loads the jQuery in index.html and in components, it can be accessible using 

`declare var jQuery: any;`

If I comment out `'jquery': 'npm:jquery/dist/jquery.js'` this from systemjs.config.js, I start getting errors in my console on `require('jquery')` and If I don't and keep both jquerys, the application breaks on quite a lot of places that are using bootstrap and other jquery plugins. Is there a workaround for this issue?

**Comment by evansmwendwa** [comment]
Try if the following works for you.

Load jQuery in your application's index.html
``` html
<script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
```

and map the same file in your systemjs.config file
``` javascript
'ng2-daterangepicker': 'npm:ng2-daterangepicker@2.0.2',
'bootstrap-daterangepicker': 'npm:bootstrap-daterangepicker@2.1.25/daterangepicker.js',
'jquery': 'https://code.jquery.com/jquery-2.2.0.min.js',
```
Then try using jQuery with declare var
``` javascript
declare var jQuery: any;
```

You might also need to add any plugins that you are using in your `customtypings.d.ts` file e.g

``` javascript
interface JQuery {
    daterangepicker(target?:any, callback?:any): JQuery;
}
```

Another options would be to replace `declare var jQuery: any;` in your code with `import * as $ from "jquery";`

If that fails post a screenshot of your browser console I have a look at the detailed logs

**Comment by bitf09a013** [comment]
I did all of above. It works but still it breaks other pages.

![3](https://cloud.githubusercontent.com/assets/5657544/23912567/e46769fc-0901-11e7-9cdc-4776ccf6a922.PNG)
![2](https://cloud.githubusercontent.com/assets/5657544/23912568/e49d300a-0901-11e7-85a2-0013915d6a0b.PNG)
 
Is there a way to use the jQuery object if its already loaded in index.html instead of requiring to install the node packages for this?

**Comment by evansmwendwa** [comment]
@bitf09a013 I managed to put up a working example using the owl carousel plugin and the ng2-daterangepicker on this plunker: https://embed.plnkr.co/ScMfJpJmTjnXKrhDaKaN/

Note the extra meta in systemjs config and also that I did not load jquery or the owl carousel plugin in the html page.

``` javascript
meta: {
    'https://code.jquery.com/jquery-3.1.1.min.js': {
        format: 'global'
    },
    'https://cdn.rawgit.com/OwlCarousel2/OwlCarousel2/develop/dist/owl.carousel.min.js': {
      format: 'global'
    }
  }
```
Also inside the component I am importing both jquery and owl on top of declaring it. The two imports are to ensure the jquery plugin is registered well after jquery has already loaded.

``` javascript
declare var $:any;
import  "jquery";
import 'owl';
```

Go through the code and see if it aids you in making your project work.
